watchMeTests/navigationOnCategoryPages.test.js

```diff
@@ -0,0 +1,31 @@
+const {snippetHomepageOpened} = require('./common');
+const suitest = require('suitest-js-api');
+const {assert, VRC, PROP, COMP, VALUE} = suitest;
+const {pictureBtnFocused, watchmeBtnFocused, pageContent} = require('./elements');
+const {snapshotElement} = require('./utils');
+
+
+describe('Navigation on Category Pages', async() => {
+	before(async() => {
+		await assert.openApp();
+		await snippetHomepageOpened();
+	});
+
+	it(`Check if the navigation through the categories in the menu bar functions correctly
```

Comment:
I would keep `it` short, in this case maybe `it('Works as expected, ...`. And the text could go to comment line inside `it`.